user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Adding_related_parameters_to_a_trend_graph.md

```diff
@@ -42,7 +42,9 @@ The following prerequisites are **optional** but highly recommended, as they wil
 If both optional requirements are met, DataMiner can study the system more efficiently, leading to even more valuable insights.
 
 > [!NOTE]
-> When you have activated this feature, it may take up to a week before the first results are available. During this time, DataMiner will analyze your system in order to learn which parameters are related.
+>
+> - You can check if any of these (optional) requirements are still missing in your system by checking the logs via *System Center* > *Logging*.
```

Comment:
Would it be an option to be more specific about which logging exactly should be checked? There's a lot of logging on the mentioned page, so the current instructions might leave the user a little lost.